### DISCUSSION TITLE
Welcome message displayed every time I start Docker container

### DISCUSSION THREAD

**Discussion by the0neWhoKnocks** [cypress-io/cypress#27987] [Questions and Help] [answered]
I've seen [discussion 1](https://github.com/cypress-io/cypress/discussions/20780) and [discussion 2](https://github.com/cypress-io/cypress/discussions/23051), and I haven't found a solution in the docs.

I run `v13.3.0` with `open --e2e --browser electron --project .`, and every time I'm presented with the "What's New in Cypress" screen, which can delay the start of me running tests because it's loading an external page with a video. I can understand if it does this once when a new version is run, but every time is too much.

Is there a way to bypass that screen, or does Cypress remember that this screen was viewed and I just need to persist that data via a Docker mount or something?

![2023-10-05_10-54 - cypress whats new](https://github.com/cypress-io/cypress/assets/344140/88b28b1e-5ea8-43fe-8948-921328f71359)

**Comment by jennifer-shehane** [comment]
It's an attribute we keep track of in the AppData called majorVersionWelcomeDismissed, with an object where 13 would track when it was dismissed. I'd check your local AppData for a global state.json file that contains this info and maybe copy that to the docker container. Haven't tried this though. 

Open Cypress via cypress open
Go to Developer Tools -> View App Data

**Reply by the0neWhoKnocks** [reply]
Hm, when I click on `View App Data` the container crashes with this error:
```
Exited with code 4
Error: Exited with code 4
    at ChildProcess.<anonymous> (<embedded>:3180:421654)
    at Object.onceWrapper (node:events:628:26)
    at ChildProcess.emit (node:events:513:28)
    at maybeClose (node:internal/child_process:1091:16)
    at Socket.<anonymous> (node:internal/child_process:449:11)
    at Socket.emit (node:events:513:28)
    at Pipe.<anonymous> (node:net:322:12)
```

**Reply by the0neWhoKnocks** [reply]
Do you have an idea of what the file path for `state.json` is within the Docker image?

**Reply by the0neWhoKnocks** [reply, marked as answer]
After digging around other posts, some trial and error, I ended up with this:
```js
// [state.json]

{
  "majorVersionWelcomeDismissed": {
    "13": 1696614696387
  }
}
```
```dockerfile
# [Dockerfile]

# Added at bottom of e2e specific Dockerfile to ensure quick builds in case any other state changes are required in the future.
COPY --chown=node:node ./state.json /home/node/.config/Cypress/cy/production/projects/__global__/
```

Ideally this should be configurable via an environment variable. It's not great that a User has to build a custom image just to disable this.
